### ISSUE TITLE
Add values content to Home and/or marketing pages

### ISSUE THREAD

**Issue by YedidaZalik** [cds-snc/notification-planning#1549]
As a potential user of Notify, I need to understand that Notify is committed to creating an accessible and equitable experience for users and recipients so that I can decide whether to use Notify. 

WHY are we building?

Related to [Line on home page could confuse users about accessibility of Notify emails](https://github.com/orgs/cds-snc/projects/40/views/5?pane=issue&itemId=60752264)#1548 . We had to remove reference to emails being "accessible" because may give users incorrect impression that emails sent from Notify are always fully accessible however screen readers do not always read text in the correct language (WCAG 3.1.2 Language of Parts); NVDA announces extra spaces in Microsoft Outlook and  Headings within the email start at h2 instead of h1 (WGAG 4.1.2 Name, Role, Value)

We strive for an accessible experience and now there is no reference to that on the Home page. 

WHAT are we building?

A more general statement on our home page about our commitment to accessibility that links to our Ally statement.

VALUE created by our solution

Potential users will understand the emphasis we put on Ally.

## Documentation and Artifacts
Good docs, figma mockups, ADRs, screenshots etc.

## Acceptance Criteria
Given some context, when (X) action occurs, then (Y) outcome is achieved 

- [ ] Cypress UI tests if needed.
- [ ] Generate appropriate log messages so that executions of this feature can be tracked
- [ ] Can misuse of this feature cause harm? If yes, create an alert
- [ ] Update the status of related findings, insights, and hypotheses on the Research Airtable
- [ ] Once change/fix/feature is implemented, link relevant Airtable records to design artifacts (Figma)

* A11y

## Related [Research Airtable](https://airtable.com/appWwAmHwDLtpIyko/tbl38n7ofWYBuezFc/viwQUX3tUxbMxYS7C?blocks=hide) records
- (Ping Adrianne to add stuff here or find stuff yourself)

## QA Steps
- [ ] Tested in a realistic production scenario

## GC Articles Publish checklist
- [ ]  Supporting feature is released and unflagged
- [ ]  Create or update slug
- [ ]  Page content and slug is translated
- [ ]  Document links to and from the page
- [ ]  Add page slug and title to [inventory](https://docs.google.com/document/d/1N6B4fSwoM40m-pDLI70BvLfL7O0n04ndyAXYaIwNbNw/edit)
- [ ]  If new page, add WAF rule
- [ ]  Create new page or update existing page content
- [ ]  Create or update translation to the page
- [ ]  Link translations by choosing an assigned translation
- [ ]  If new page, save as draft
- [ ]  If existing page, update the page
- [ ]  Post page in Slack for last minute reviews
- [ ]  On staging: Clear the cache through GC Notify's admin panel
- [ ]  On staging: Check links, images and language toggle
- [ ]  Give a heads up to the app-team channel once ready to publish
- [ ]  On staging and production: Clear the cache through GC Notify's admin panel
- [ ]  Clear Cloudfront cache in GCA as needed (Andrew and Phil are GCA admins)

**Comment by connorscarolyns** [comment]
I'm also ok with closing this ticket if it's too. much, and we can reopen a new one in the future if this arises again.

**Comment by YedidaZalik** [comment]
@connorscarolyns my suggestion is to add this to the A11y feedback mechanism work and we can update the public pages with content about A11y when we launch that.

In terms of the wording, I'm hesitant to use "helping public servants send accessible messages" because while the product itself is accessible to public servants, we don't know if the messages are accessible. We have the following heading in the A11y statement:

"This statement reflects the accessibility of our client interface, not messages received by end users"

That section goes on to state:

"Your email recipients might use additional software and assistive technology to read messages from GC Notify. Their software and technology could impact the accessibility of your messages. For example:

- In Microsoft Outlook, NonVisual Desktop Access (NVDA) announces extra spaces. 
- Some technologies do not read text in the correct language."

**Comment by connorscarolyns** [comment]
I disagree here. "Helping public servants send accessible messages" does not say that the messages are 100% perfect. The fact is, when it comes to accessibility, it's almost impossible to be 100% perfect, too many tools with conflicting functionality. We can also point to the known issues for clarity. But we do HELP public servants send accessible messages.
